lib/collection/request-auth.js

```diff
@@ -131,122 +23,96 @@ _.inherit((
     RequestAuth = function PostmanRequestAuth (options) {
         // this constructor is intended to inherit and as such the super constructor is required to be executed
         RequestAuth.super_.call(this, options);
-        if (!options) { return; } // in case definition object is missing there is no point moving forward
 
-        _.assign(this, /** @lends RequestAuth.prototype */ {
-            /**
-             * @type {String|undefined}
-             */
-            type: undefined
-        });
+        if (_.has(options, 'type')) {
+            this.use(options.type);
+        }
 
         // load all possible auth parameters from options
         _.forEach(_.omit(options, 'type'), this.update.bind(this));
-        this.use(options.type);
     }), Property);
 
 _.assign(RequestAuth.prototype, /** @lends RequestAuth.prototype */ {
     /**
      * Update the parameters of a specific authentication type. If none is provided then it uses the one marked as to be
      * used.
      *
-     * @param {Object} options
-     * @param {AuthTypes=} [type]
+     * @param {VariableList|Array|Object} options
+     * @param {String=this.type} [type]
```

Comment:
I might be wrong here but, isn't the correct way is 
```js
// @param {String} [type=this.type]
```
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values